test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -2233,6 +2234,8 @@
     (ca/overwrite-existing-crl! crl rest-of-full-chain (:capub settings) (:cakey settings) ca-cert (vec all-serials) (:cacrl settings))
     (let [[updated-crl & _rest-of-full-chain] (utils/pem->crls (:cacrl settings))
           updated-crl-number (utils/get-crl-number updated-crl)]
+      (is (= (.getName (.getIssuerX500Principal ^X509CRL updated-crl))
```

Comment:
This tests the defect.  Without the change to use the issuer from the crl in `certificate_authority` this test fails.